workflows/ncn/hooks/before-each/cos-prechecks-for-worker-reboots.yaml

```diff
@@ -34,7 +34,11 @@ spec:
     SSH_OPT="${SSH_OPT} -o ConnectTimeout=10"
     # Note: this file path cannot change without coordinated changes to the COS
     # product stream
-    file_path=/opt/cray/shasta/cos/bin/prechecks_for_worker_reboots
+    if [[ -d /opt/cray/shasta/cne ]];then
```

Comment:
Looking at the code closer it seems like we'd need to check for the path on the ${TARGET_NCN} rather than locally

Reply:
Good catch.  Let me thunk on that.

Reply:
pre updated and new logic tested